GoogleForCreators/web-stories-wp
# Fix authors loading + Convert FontPicker to a general component.

## Summary
Fixes authors input loading all authors and displays only everyone with/above author rights only.
Display max 100 authors as visible options and allow searching among all.
Converts FontPicker to be a general component and use it for authors as well.


## Relevant Technical Choices



## To-do

- [x] Load only the first 100 authors as primary options.
- [x] Better naming instead of DropDown2



## User-facing changes



## Testing Instructions
- Add some subscribers as users.
- Add a story and open the document tab
- Verify that your current user is displayed as the author of the story.
- Click to change the author, verify that the subscribers are not visible in the user list
- Verify that it's possible to search among the authors.
- Verify that the author is changed correctly when changing.


---



Fixes #4749
